README.md

```diff
@@ -1,36 +1,232 @@
 # Twilio Breakout Massive SDK for Arduino
 
-Breakout SDK enables developers to exchange data between low-powered devices and IoT cloud services. Breakout makes it easy to adopt low power optimizations provided by the Narrowband network and cellular modem hardware.
+Breakout SDK enables developers to connect their devices to an MQTT broker over Narrowband cellular line.
 
-This SDK is primary intended for use with our Massive IoT Developer Kits but can likely also be used on similar boards with Arduino support with modification.
+This SDK is primary intended for use with our [Massive IoT Developer Kits](https://www.twilio.com/docs/wireless/quickstart/broadband-iot-developer-kit) and [Twilio Narrowband SIMs](https://www.twilio.com/wireless/narrowband) but can likely also be used on similar boards with Arduino support with modification.
 
-The current Twilio Massive IoT Developer Kit, version "Alfa" is based on a Wio factor development board with numerous Grove connectors and a Narrowband capable ublox SARA-N410 cellular module.  The developer kit has thusfar been distributed at [SIGNAL](https://www.twilio.com/signal) and comes with Grove sensors – humidity, light, and ultrasonic distance.
+This Arduino compatible library is backed by a more generic cellular module abstraction layer, [Twilio Breakout Massive SDK](https://github.com/twilio/Breakout_Massive_SDK), which also has support for the Quectel BG96 NB-IoT module.  The goal of this module is to make packet based communications easier when dealing with these cellular modules.
 
-It is important to note that you must have a Narrowband SIM to utilize this kit, a standard LTE SIM will not be able to connect to the network.
```

Comment:
It's worth keeping a call-out here that Narrowband SIMs are required with the Massive IoT Developer kit since this is not always clear for users.

Reply:
Narrowband SIMs are mentioned a line above. Does it feel ambiguous?